### PR TITLE
Aidan/v10 upgrade handler claim param migration

### PR DESCRIPTION
**Pull request by asalzmann**
**Summary**
As a safety precaution, migrate claim distributor addresses to a more secure key setup.

**Test plan**
* Unittests 
* Integration tests

For integration tests, I tested the following cases 
* on commit `bdb7a1`
    * setup airdrop
    * upgrade
    * claim
```
Initial balance before claim [1000000000ustrd expected]:
amount: "1000000000"
denom: ustrd

Claiming free amount...
  code: 0
  txhash: DC2A7A05EA95D4731B1E1D71F3E6515653BE1E3ABEB289AADC9C570785DA9CE5

Balance after claim [1000120000ustrd expected]:
amount: "1000120000"
denom: ustrd

Staking...
  code: 0
  txhash: D55B72B659403D26EB6E63A4314AB84D3C2C028B897812F793B5DBFFA8FE11AD

Balance after stake [1000239900ustrd expected]:
amount: "1000239900"
denom: ustrd

Liquid staking...
  code: 0
  txhash: 942E2F298916BB8287A699272995819A3F440A6DD3C5C741E51D47E774533277

Balance after liquid stake [1000599900ustrd expected]:
amount: "1000599900"
denom: ustrd
```

* on commit `1553d4`
    * setup airdrop
    * partially claim from an account
    * upgrade
    * claim the rest from an account
```
Staking...
  code: 0
  txhash: D55B72B659403D26EB6E63A4314AB84D3C2C028B897812F793B5DBFFA8FE11AD

Balance after stake [1000239900ustrd expected]:
amount: "1000239900"
denom: ustrd

Liquid staking...
  code: 0
  txhash: 942E2F298916BB8287A699272995819A3F440A6DD3C5C741E51D47E774533277

Balance after liquid stake [1000599900ustrd expected]:
amount: "1000599900"
denom: ustrd
```

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	path = deps/hermes
 	url = https://github.com/informalsystems/ibc-rs.git
 [submodule "deps/relayer"]
-	# Commit: v2.3.1
+	# Commit: e95dd80608536c31d37354bdd7f7ec46a2172009
```
Comment (asalzmann) [inline .gitmodules:6]: revert

**Comment by sampocs** [comment]
@asalzmann can you pull in the latest changes from the v10-upgradeh=-handler branch?

File: app/upgrades/v10/upgrades.go [hunk 2]
```diff
@@ -47,13 +49,31 @@ var (
 	CommunityPoolGrowthProportion         = "0.2158"
 	StrategicReserveProportion            = "0.4879"
 	CommunityPoolSecurityBudgetProportion = "0.1358"
+	StrdDenom                             = "ustrd"
 
 	CommunityPoolGrowthAddress = "stride1lj0m72d70qerts9ksrsphy9nmsd4h0s88ll9gfphmhemh8ewet5qj44jc9"
 	BadKidsCustodian           = "stride17z6yy8vfgklgej9m848jm7rkp270gd9pgaw8zu"
 	BadKidsTransferAmount      = sdk.NewInt(15_000_000_000)
 	Ustrd                      = "ustrd"
```
Comment (sampocs) [inline app/upgrades/v10/upgrades.go:57]: nit: remove one of the Strd constants

**Review by sampocs — APPROVED** [review]
lgtm

File: dockernet/scripts/airdrop/airdrop1_standard.sh [hunk 3]
```diff
@@ -42,6 +42,8 @@ sleep 5
 $STRIDE_MAIN_CMD tx claim set-airdrop-allocations gaia stride1nf6v2paty9m22l3ecm7dpakq2c92ueyununayr 1 --from distributor-test -y | TRIM_TX
 sleep 5
 
+WAIT_FOR_STRING $STRIDE_LOGS "v10 Upgrade Complete"
```
[the remark this answers is not in the document]
Comment (asalzmann) [reply dockernet/scripts/airdrop/airdrop1_standard.sh:45]: I'm working through integration tests on different commits

**Pull request merged**